README.md

```diff
@@ -1 +1 @@
-Najja J. Marshall's pipeline  - Neural control of movment
+Datajoint pipeline for the Churchland Lab
```

Comment:
It will be helpful to include the schema diagrams in the README with a brief explanation of the overall design.